Update to Html(home, members, trainers) Python(sql-commands))

CineraM opened the pull request:
Finished implementing the home page.
Added functionality to the delete button of the members page (deletes member from the DB).
Created the html file for the trainers page.
All the corresponding sql functions were added to sql-commands

CineraM committed:
Update to Html(home, members, trainers) Python(sql-commands))

Finished implementing the home page.
Added functionality to the delete button of the members page (deletes member from the DB).
Created the html file for the trainers page.
All the corresponding sql functions were added to sql-commands

CineraM committed:
update

kbsanchez commented on templates/trainer.html:
```diff
@@ -0,0 +1,36 @@
+      <td>{{ trainer[4] }}</td>
+      <td>
+      <td>
+      <button class="btn btn-outline btn-info">Something ? </button>
```
What is the functionality of this button?

CineraM replied:
Place holder button for future functionality

kbsanchez reviewed (approved):
LGTM

CineraM committed:
Code-Update

Search bar for table pages (Classes, Trainers, Members).
Date-Time input for register as a member.
Image to home-page

CineraM committed:
Update README.md

CineraM committed:
Update README.md

kbsanchez committed:
Merge branch 'main' into Code-Updates-Matias
